src/Data/Options.purs

```diff
@@ -1,3 +1,5 @@
+-- | ## Overview of the Problem
```

Comment:
I don't actually know if markdown headers render in module docs. Have you been able to test that out?

Reply:
They work. I just tried it on my learning repo via `spago docs`:
![whether headers work](https://user-images.githubusercontent.com/8413037/93255533-eafa3f80-f74e-11ea-9efe-42d920ae6e58.png)